src/components/common/Detail.vue

```diff
@@ -36,34 +36,24 @@
 <script lang="ts">
 import { Component, Mixins, Prop, Watch } from 'vue-property-decorator'
 import { DetailComment as DetailCommentShared } from '@bcrs-shared-components/detail-comment/'
-import { Action, Getter } from 'vuex-class'
+import { Action } from 'vuex-class'
 import { ActionBindingIF } from '@/interfaces/'
-import { DateMixin } from '@/mixins'
+import { FilingTemplateMixin } from '@/mixins'
 
 @Component({
   components: { DetailCommentShared }
 })
-export default class Detail extends Mixins(DateMixin) {
+export default class Detail extends Mixins(FilingTemplateMixin) {
```

Comment:
The necessary getters, and the default correction detail comment, are inherited from the Filing Template mixin.